src/tools/github2gitlab.rst

```diff
@@ -0,0 +1,73 @@
+=====================================
+Transition your Github Repo to Gitlab
+=====================================
+
+Create Gitlab Account
```

Comment:
There is also a section on using Github in the git usage page that deals with account creation. These need to be reconciled in order to avoid duplication. WE can keep info just in one single page and link to this when necessary.